src/ucs/profile/profile_on.h

```diff
@@ -117,9 +117,11 @@ BEGIN_C_DECLS
     \
     _ret_type _name(__VA_ARGS__) { \
         UCS_PROFILE_SCOPE_BEGIN(); \
-        _ret_type _ret = _name##_inner _arglist; \
-        UCS_PROFILE_SCOPE_END(#_name); \
-        return _ret; \
+        { \
+            _ret_type _ret = _name##_inner _arglist; \
```

Comment:
maybe just define `_ret_type _ret` in the begining of the function, no need to open new {} scope?